plugins/task/src/index.ts

```diff
@@ -14,14 +14,29 @@
 //
 
 import { plugin } from '@anticrm/platform'
-import type { Asset } from '@anticrm/status'
+import type { Asset, IntlString } from '@anticrm/status'
 import type { Plugin, Service } from '@anticrm/platform'
-import type { Space } from '@anticrm/core'
+import type { Account, Doc, Ref, Space } from '@anticrm/core'
 
 export interface Project extends Space {}
 
 export interface TaskService extends Service {}
 
+export const TaskStatuses = {
```

Comment:
Why not use regular enum with string values?

Reply:
I think extend it additional fields (color as example)